src/model/RecentlyCommentedAsset.ts

```diff
@@ -7,6 +7,7 @@ const ctx = {
     editor: VocabularyUtils.PREFIX + "má-editora",
     vocabulary: VocabularyUtils.IS_TERM_FROM_VOCABULARY,
     lastComment: VocabularyUtils.JE_TEMATEM,
+    lastMyComment: VocabularyUtils.MA_MUJ_POSLEDNI_KOMENTAR,
```

Comment:
Context property does not match the property defined in the `RecentlyCommentedAssetData` interface.